astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -126,7 +165,10 @@ def icrs_to_gcrs(icrs_coo, gcrs_frame):
         pv = np.rollaxis(np.rollaxis(pv, 0, pv.ndim), 0, pv.ndim)
 
     jd1, jd2 = get_jd12(gcrs_frame.obstime, 'tdb')
-    astrom = erfa.apcs13(jd1, jd2, pv)
+    earth_pv, earth_heliocentric = prepare_earth_position_vel(gcrs_frame.obstime)
+
+    # get astrometry context
```

Comment:
The "get astrometry context"? If so, no, that's a reference to the ```astrom``` object. I've made the comments in this section clearer.